src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-credentialsId.html

```diff
@@ -1,9 +1,11 @@
 <div>
-    <p>Jenkins requires credentials to authenticate with Bitbucket. They give Jenkins access to some or all of the repositories on your Bitbucket instance and allow it to check out source code for building.
-        Here you can:</p>
+    <p>When running a job, Jenkins will use these credentials to check out the source code for builds. For Jenkins to
+        do this, it needs credentials with access to the projects and repositories you want it to build from.</p>
+
+    <p>To provide Jenkins with credentials you can:</p>
     <ul>
-        <li>use the dropdown list to select the credentials you want Jenkins to use, or</li>
-        <li>leave 'none' selected to use the instance defined credentials (if they exist).</li>
+        <li>Choose credentials from the list.</li>
+        <li>Add credentials as an SSH username and private key.</li>
```

Comment:
```suggestion
        <li>Add credentials as a Username with password.</li>
```
We do not support SSH at this stage.